build.gradle.kts

```
@@ -1,15 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-
     repositories {
         google()
         jcenter()
+        mavenCentral()
```

I don't think this addition does anything since jcenter will look for things on mavenCentral as well. I usually declare these in the opposite order, so that if something I'm looking for on maven is shadowed incorrectly on jcenter, I get the correct one.
```suggestion
        mavenCentral()
        jcenter()
```